tests/acl/custom_acl_table/test_custom_acl_table.py

```diff
@@ -0,0 +1,274 @@
+import logging
+import json
+import pytest
+import time
+
+from ptf.mask import Mask
+import ptf.packet as scapy
+
+
+import ptf.testutils as testutils
+from tests.common.utilities import skip_release
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology("t0"), # Only run on T0 testbed
+    pytest.mark.disable_loganalyzer,  # Disable automatic loganalyzer, since we use it for the test
+]
+
+CUSTOM_ACL_TABLE_TYPE_SRC_FILE = "acl/custom_acl_table/custom_acl_table.json"
+CUSTOM_ACL_TABLE_TYPE_DST_FILE = "/tmp/custom_acl_table.json"
+
+ACL_RULE_SRC_FILE = "acl/custom_acl_table/acl_rules.json"
+ACL_RULE_DST_FILE = "/tmp/acl_rules.json"
+
+LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
+LOG_EXPECT_ACL_RULE_FAILED_RE = ".*Failed to create ACL rule.*"
+
+
+@pytest.fixture(scope='module', autouse=True)
+def check_release(rand_selected_dut):
+    skip_release(rand_selected_dut, ["201811", "201911", "202012"])
+
+
+@pytest.fixture(scope='module')
+def setup_counterpoll_interval(rand_selected_dut):
+    """
+    Set the counterpoll interval for acl to 1 second (10 seconds by default)
+    """
+    # Set polling interval to 1 second
```

Comment:
@bingwang-ms This comment is 1 s, but in code it's 1000. Should update comment?

Reply:
No, cause the polling interval for counter is in ms, that's why the code is 1000.
See this
```
admin@str2-7050cx3-acs-10:~$ counterpoll acl interval 1
Usage: counterpoll acl interval [OPTIONS] POLL_INTERVAL
Try "counterpoll acl interval --help" for help.

Error: Invalid value for "POLL_INTERVAL": 1 is not in the valid range of 1000 to 30000.
```